concepts/50 React Components/40 Component Configuration Syntax/55 Markup Customization/0 Markup Customization.md

```diff
@@ -1,5 +1,5 @@
-Templates allow you to customize UI components. In the DevExtreme API, template properties end with `Template`: `itemTemplate`, `groupTemplate`, `contentTemplate`. When you specify them in React, replace `Template` in the name with `Render` or `Component`, depending on whether the template is a rendering function or custom component. For example, instead of `itemTemplate`, use `itemRender` or `itemComponent`.
+Templates allow you to customize UI components. In the DevExtreme API, template properties end with `Template`: `itemTemplate`, `groupTemplate`, `contentTemplate`. When you specify them in React, replace `Template` in the name with `Render` or `Component`, depending on whether the template is a rendering function or custom component. For example, instead of `itemTemplate`, use `itemRender` or `itemComponent`. If a component does not reinitiate a template when the component rerenders, you should use the `itemComponent` property. Otherwise, we recommend you to use the `itemRender` property. 
```

Comment:
```suggestion
Templates allow you to customize UI components. In the DevExtreme API, template properties end with `Template`: `itemTemplate`, `groupTemplate`, `contentTemplate`. When you specify them in React, replace `Template` in the name with `Render` or `Component`, depending on whether the template is a rendering function or custom component. For example, instead of `itemTemplate`, use `itemRender` or `itemComponent`. If a component does not reinitiate a template when the component rerenders, you should use the `itemComponent` property. Otherwise, we recommend that you use the `itemRender` property. 
```